capstone/src/main/webapp/index.html

```diff
@@ -2,11 +2,11 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Club Hub</title>
+    <title>ClubHub</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    hello
+    <button class="profile-button" onclick="location.href='profile.html';">My Profile</button>
```

Comment:
Isn't this basically what's going to be in the navbar?

Reply:
Yep! I wanted to wait for all the files to be in master before I created the separate navbar - it should be in the next PR